utilities/validations.py

```diff
@@ -133,6 +133,22 @@ def validate_structure_id(**kwargs):
     table
     :param structure_id
     """
+    if(kwargs.get('unique_structure_id')):
+        structure_id_list = [structure['structure_id']
```

Comment:
Avoid using a list comprehension, let's have a more readable code. Structure id should be validated by the ids in the structure model.

Reply:
Updated.